pkg/entities/vault.go

```diff
@@ -19,10 +19,32 @@ import (
 )
 
 func (e *Entity) CreateVault(req *request.CreateVaultRequest) (*model.Vault, error) {
+	// auto generate vault address when desig mode = false
+	walletAddress := ""
+	walletNumber := -1
```

Comment:
in case desig mode = true, we wont create wallet and will store -1 in database to differentiate with others
wallet number > 0 when desig mode = false